src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -86,90 +121,140 @@ export default class RiddleOfFire extends Module {
+				unless you need to hold a charge for strategic purposes.
 			</Trans>,
-			matcher: matchClosestHigher,
 			tiers: {
-				1: SEVERITY.MEDIUM,
-				2: SEVERITY.MAJOR,
+				1: SEVERITY.MINOR,	// Always a minor suggestion
 			},
-			value: missedGcds,
-			why: <Trans id="mnk.rof.suggestions.gcd.why">
-				<Plural value={missedGcds} one="# GCD was" other="# GCDs were" /> missed during RoF.</Trans>,
+			value: riddlesWithOneTackle,
+			why: <Trans id="mnk.rof.suggestions.tackle.why">
+				<Plural value={riddlesWithOneTackle} one="# use" other="# uses" /> of <StatusLink {...STATUSES.RIDDLE_OF_FIRE} /> contained
+				only one use of <ActionLink {...ACTIONS.SHOULDER_TACKLE} />.
+			</Trans>,
 		}))
 	}
 
 	private stopAndSave(endTime: number = this.parser.currentTimestamp): void {
 		if (this.riddle && this.riddle.active) {
-			this.history.push({...this.riddle, active: false, end: endTime})
-		}
-	}
-
-	private getGcdCount(riddle: Riddle): number {
-		return riddle.casts.filter(cast => {
-			const action = getDataBy(ACTIONS, 'id', cast.ability.guid) as TODO
-			return action && action.onGcd
-		}).length
-	}
-
-	private formatGcdCount(count: number): JSX.Element {
-		if (count <= ROF_GCD.ERROR) {
-			return <span className="text-error">{count}</span>
-		}
+			this.riddle.active = false
+			this.riddle.end = endTime
 
-		if (count === ROF_GCD.WARNING) {
-			return <span className="text-warning">{count}</span>
+			this.history.push(this.riddle)
 		}
-
-		return <span className="text-success">{count}</span>
 	}
 
 	output() {
-		const panels = this.history.map(riddle => {
-			const numGcds = this.getGcdCount(riddle)
-
-			return {
-				key: riddle.start,
-				title: {
-					content: <>
-						{this.parser.formatTimestamp(riddle.start)}
-						<span> - </span>
-						<Trans id="mnk.rof.table.gcd" render="span">
-							{this.formatGcdCount(numGcds)} <Plural value={numGcds} one="GCD" other="GCDs" />
-						</Trans>
-						<span> - </span>
-						{riddle.rushing && <>
-							&nbsp;<Trans id="mnk.rof.table.rushing" render="span" className="text-info">(rushing)</Trans>
-						</>}
-					</>,
+		return <RotationTable
+			targets={[
+				{
+					header: <Trans id="mnk.rof.table.header.gcds">GCDs</Trans>,
+					accessor: 'gcds',
+				},
+				{
+					header: <ActionLink showName={false} {...ACTIONS.ELIXIR_FIELD}/>,
+					accessor: 'elixirField',
 				},
-				content: {
-					content: <Rotation events={riddle.casts}/>,
+				{
+					header: <ActionLink showName={false} {...ACTIONS.SHOULDER_TACKLE}/>,
+					accessor: 'shoulderTackle',
 				},
+			]}
+			data={this.history
+				.map(riddle => ({
+					start: riddle.start - this.parser.fight.start_time,
+					end: riddle.end != null ?
+						riddle.end - this.parser.fight.start_time
+						: riddle.start - this.parser.fight.start_time,
+					targetsData: {
+						gcds: {
+							actual: riddle.gcds,
+							expected: EXPECTED_GCDS,
```

Comment:
Making expected GCDs a field in the Riddle the same way AoE uses targets might be a good idea. This is so we can show the correct expectation for opener and rushing Riddles and avoid false-flagging every opener. Depends on either using the new core gauge, or seeing if non-RoW was in use. Could probably do it in the stopAndSave function.